### PR TITLE
758 FDC3 2.0 docs release

### LINKED ISSUE #758: Create 2.0 version in microsite and fresh next draft
As part of the FDC3 2.0 release, create a 2.0 version, promote to current, prepare next for 2.1 additions.

### PR DESCRIPTION
**Pull request by kriswest**
resolves #758 by creating a new version in the site and updating the version used /docs/fdc3-[intro/standard].md for 2.1

### CODE REVIEW

**Review by greyseer256 — APPROVED** [review]
Minor comments, looks good 👍

File: website/versioned_docs/version-2.0/api/spec.md [hunk 1]
```diff
@@ -0,0 +1,580 @@
...
+];
+```
+
+### Direct Listening and Broadcast on Channels
+
+While joining User channels (using fdc3.joinUserChannel) automates a lot of the channel behaviour for an app, it has the limitation that an app can only be 'joined' to one channel at a time.  However, an app may instead retrieve a `Channel` Object via the [`fdc3.getOrCreateChannel`](ref/DesktopAgent#getorcreatechannel) API, or by raising an intent that returns a channel. The `Channel` object may then be used to listen to and broadcast on that channel directly using the [`Channel.addContextListener`](ref/Channel#addcontextlistener) and the [`Channel.broadcast`](ref/Channel#broadcast) APIs. This is especially useful for working with dynamic *App Channels*. FDC3 imposes no restriction on adding context listeners or broadcasting to multiple channels.
+
+### App Channels
+
+App Channels are topics dynamically created by applications connected via FDC3. For example, an app may` create a channel to broadcast to others data or status specific to that app.
```
[the remark this answers is not in the document]
Comment (kriswest) [reply website/versioned_docs/version-2.0/api/spec.md:525]: Looks likely Also thats a terrible sentence!

Going to replace with:
>App Channels are topics dynamically created by applications connected via FDC3. For example, an app may create a named App Channel to broadcast data or status that is specific to that app to other apps that know to connect to the channel with that name.

Its still not wonderful prose but is I hope easier to parse

File: website/versioned_docs/version-2.0/api/ref/Metadata.md [hunk 2]
```diff
@@ -0,0 +1,396 @@
...
+```ts
+interface ContextMetadata {
+  /** Identifier for the app instance that sent the context and/or intent. 
+   *  @experimental 
+   */
+  readonly source: AppIdentifier;
+}
+```
+
+Metadata relating to a context or intent & context received through the `addContextListener` and `addIntentListener` functions. Currently identifies that originated the context or intent message.
```
[the remark this answers is not in the document]
Comment (kriswest) [reply website/versioned_docs/version-2.0/api/ref/Metadata.md]: went with the latter suggestion (as it'll have an `appId` we can always call it an app)

**Comment by kriswest** [comment]
> Minor comments, looks good 👍

Thanks @greyseer256, some good catches

**Review by openfin-johans — APPROVED** [review]
Ship it captain!

**Pull request merged**